### ISSUE TITLE
Hi, you are missing a few key cartographic icons such as:

### ISSUE THREAD

**Issue by spydmobile** [mapshakers/mapkeyicons#9]
I noticed you have icons like: Magic Portal and have more than one castle, as a serious mapplication developer, I would love to see:

- Cloud
- smoke
- weather

and for moving map.....

- helicopter (top down)
- small aircraft (top down)
- large aircraft (top down)

Thanks for your great work!

**Comment by boldtrn** [comment]
Yes, a couple more icons would be nice. Also icons like "meeting point" or "coffee" would be nice.

**Comment by stefangrotz** [comment]
Hey great work. I miss an icon for vending machines. Is this project still in active development?
